docs/site/Implementing-auto-migration.md

```diff
@@ -0,0 +1,158 @@
+---
+lang: en
+title: 'Implementing Auto-Migration'
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Implementing-auto-migration.html
+---
+
+## Overview
+
+In LoopBack, auto-migration helps the user create relational database schemas
```

Comment:
It makes me wonder what will happen if we run auto-migration in nosql database. :)